client/src/Movies/SavedList.js

```diff
@@ -10,9 +13,13 @@ export default class SavedList extends Component {
       <div className="saved-list">
         <h3>Saved Movies:</h3>
         {this.props.list.map(movie => (
+          <Link to={`/movies/${movie.id}`}>
```

Comment:
If you're up to the challenge, I suggest trying to add a feature to prevent duplicate movies from being entered.